epub34/rs/index.html

```diff
@@ -287,12 +287,11 @@ <h3>Error reporting</h3>
 					they are strongly encouraged to provide a means of accessing this information. A comparable example
 					are the developer tools that web browsers provide for debugging HTML pages and applications.</p>
 
-				<p>[=EPUB creators=], for example, can greatly benefit from having access to such processing information
-					(e.g., to efficiently debug their EPUB publications). For maximum use in debugging, it is
-					recommended that reading systems not only report issues they directly encounter but also issues
-					reported by any applications they use (e.g., HTML, CSS, and JavaScript errors reported from a
-					browser core used to render the content, or validation issues reported by <a
-						href="https://www.w3.org/publishing/epubcheck/">EPUBCheck</a>).</p>
+				<p>Access to such processing information allows, for example, more efficient debugging of EPUB
+					publications. For maximum use in debugging, it is recommended that reading systems not only report
```

Comment:
```suggestion
					publications. For maximum use in debugging, it is recommended that reading systems not only report
```

I believe "RECOMMENDED" is also an RFC term, isn't it? I am not sure what to replace it with, though. "it is preferable"?

Reply:
Ya, when I was going through the a11y docs I realized I forgot to check for recommended in these two. I was only looking for must, should, and may. Planning to do another pass later today as there are a few of these to write out. It's a synonym for "should" so I thin "advised" also works well for these and makes the guidance more consistent.

Reply:
I've updated these references in the last commit. There are a small handful that I think are fine to leave in lowercase as they're being used as adjectives that can't be substituted (but don't make normative requirements) or describing support levels in another specification. These can't be confused with requirements.